### DISCUSSION TITLE
Peaksjs Widget

### DISCUSSION THREAD

**Discussion by antoinedaurat** [bbc/peaks.js#486] [Show and tell]
Hi,

I just wanted to thank you for the amazing work you're doing on this package and let you know that I started a jupyter widget wrapping peaks.js closely for interacting with audio data in jupyter notebooks.

I had such a hard time finding a good waveform widget for python and was so thankful that I _just_ had to wrap a `peaks` instance for making a great one. I ought to let you know. And to thank you. Again! 🙏 

link to the repo: https://github.com/ktonal/peaksjs-widget

Cheers!

**Comment by chrisn** [comment]
Wow, thank you for sharing this and for your kind words. The Jupyter widget looks really good 😍
